src/server.js

```diff
@@ -64,10 +66,12 @@ app.post(`${API}/register`, (req, res) => {
 app.post(`${API}/team/:teamId/start-survey`, (req, res) => {
     const team_shortId = req.params.teamId;
     const surveyId = shortid.generate();
+    const survey_name = req.body.name;
 
-    connection.query(
-        `INSERT INTO survey (survey_uuid, team_id) 
-         SELECT '${surveyId}', team_id FROM team WHERE team.team_uuid = '${team_shortId}'`, (err, results, fields) => {
+    const query = `INSERT INTO survey (survey_uuid, survey_name, team_id) VALUES 
```

Comment:
Does this not leave us vulnerable for sql injection? Or will the mysql orm still know to prevent that?

Reply:
@wongm3 Thats actually why I added `connection.escape()` to all inputs we get. That will prevent the possibility of sql injections.

Reply:
We already know about sql injection but MVP, we will come back to it